pems/core/views.py

```diff
@@ -0,0 +1,10 @@
+from django.views.generic import TemplateView
+
+
+class PageNotFoundView(TemplateView):
+    template_name = "404.html"
+
+    def dispatch(self, request, *args, **kwargs):
+        response = super().dispatch(request, *args, **kwargs)
+        response.status_code = 404
```

Comment:
This doesn't feel right (manually setting the status code). 

I think all we need to do here is define the `404.html` template file. Django serves that by default (with a 404 status code) when a page is not found: https://docs.djangoproject.com/en/5.2/ref/views/#django.views.defaults.page_not_found

Reply:
Yep, since there is no need for it, I removed the custom view and just defined the `404.html` template file in 7fb0979a24297fb2c91aa8f87f8a527e4a9a10d4